packages/models-library/src/models_library/api_schemas_resource_usage_tracker/pricing_plans.py

```diff
@@ -30,30 +31,68 @@ class PricingUnitGet(BaseModel):
                 {
                     "pricing_unit_id": 1,
                     "unit_name": "SMALL",
-                    "unit_extra_info": UnitExtraInfo.model_config["json_schema_extra"]["examples"][0],  # type: ignore [index]
```

Comment:
consider refactoring this to callable 

SEE https://github.com/ITISFoundation/osparc-simcore/pull/7200#discussion_r1970089862